pyproject.toml

```diff
@@ -82,7 +82,7 @@ exclude = '''
     | buck-out
     | build
     | dist
-    | datasetinsights/data/datasets/protos
+    | datasetinsights/datasets/protos
```

Comment:
Can you look into using general regex pattern to ignore protobuf file? How do we make sure we don't have to manually add new protobuf files to be ignored in the futrue?

Reply:
Fixed!

Reply:
I think what YC proposes is to use a regular expression, in case we have protobuf files in another folder.